source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: true
 
 require "fluent/plugin/input"
+require_relative "oms_common"
```

Comment:
why do we need this dependency?

Reply:
`module Fluent::Plugin
  class Kube_PodInventory_Input < Input
    Fluent::Plugin.register_input("kube_podinventory", self)

    @@hostName = (OMS::Common.get_hostname)`
    
  
The call to OMS::Common.get_hostname fails without it. Earlier this plugin was not the first plugin in the conf file and by the time it started running oms_common was already imported in global context. But since now its the first plugin we need to explicity import it